test/fixtures/es-modules/loop.mjs

```diff
@@ -7,4 +7,6 @@ while (t > 0) {
     console.log(`Outputed message #${k++}`);
   }
 }
-process.exit(55);
\ No newline at end of file
+process.exit(55);
+
+(function force_context_allocation() { return t + k; })
```

Comment:
Can you add a comment explaining why this is necessary. I worry that with the rate of code changes, the git commit message may get lost.

Reply:
Done.